02_configure_host.sh

```diff
@@ -123,63 +123,64 @@ if [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
     # Adding an IP address in the libvirt definition for this network results in
     # dnsmasq being run, we don't want that as we have our own dnsmasq, so set
     # the IP address here
-    if [ ! -e /etc/sysconfig/network-scripts/ifcfg-${PROVISIONING_NETWORK_NAME} ] ; then
+    sudo nmcli con add type bridge con-name ${PROVISIONING_NETWORK_NAME} ifname ${PROVISIONING_NETWORK_NAME}
+    sudo nmcli con modify ${PROVISIONING_NETWORK_NAME} connection.zone ${ZONE}
+    sudo nmcli con modify ${PROVISIONING_NETWORK_NAME} connection.autoconnect TRUE
+    # This setting will bring the bridge ports up when the bridge itself is brought up.
+    sudo nmcli con modify ${PROVISIONING_NETWORK_NAME} connection.autoconnect-slaves 1
         if [[ "$(ipversion $PROVISIONING_HOST_IP)" == "6" ]]; then
-            echo -e "DEVICE=${PROVISIONING_NETWORK_NAME}\nTYPE=Bridge\nONBOOT=yes\nNM_CONTROLLED=no\nIPV6_AUTOCONF=no\nIPV6INIT=yes\nIPV6ADDR=${PROVISIONING_HOST_IP}/64${ZONE}" | sudo dd of=/etc/sysconfig/network-scripts/ifcfg-${PROVISIONING_NETWORK_NAME}
+	    sudo nmcli con modify ${PROVISIONING_NETWORK_NAME} ipv6.address $PROVISIONING_HOST_IP/64
+	    sudo nmcli con modify ${PROVISIONING_NETWORK_NAME} ipv6.addr-gen-mode stable-privacy
```

Comment:
Given that the bridge is statically configured, so we need the addr-gen-mode?

Not blocking, just wondered why it was added - currently we only support DHCPv6 for ipv6 testing via dev-scripts, and we configure the [libvirt DHCP reservations](https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/libvirt/templates/network.xml.j2#L93) so the IPs are predictable (relatedly we configure the target cluster hosts to have a [predictable DUID/IAID](https://github.com/openshift/machine-config-operator/blob/master/templates/common/on-prem/files/NetworkManager-onprem.conf.yaml#L9..L10))

Reply:
I am glad you asked, as I was unsure when I put this in. My goal was to get something working on my local machine and I was hitting a lot of "gotchas" about the order of the commands and what not. so I will test with this line removed to make sure it still works.